templates/example_mixing_score.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>The user isn't running any of these notebooks here. If someone has just stumbled upon this notebook, the description should give more guidance about what we're expecting them to do. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>